rekolekcje-engine/src/verify/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/storage/ParticipantData.java

```diff
@@ -31,8 +31,13 @@
 
   // health report
   private final String currentTreatment;
+  private final String mentalDisorders;
   private final String medications;
   private final String allergies;
+  private final String medicalDiet;
+  private final Boolean mountainWalkingContraindications;
```

Comment:
too complicated (remember KISS ;) ). Spróbuj: `canHike`, `canHikeInMountains`, `canGoHiking` albo coś podobnego. Dobrze, żeby nazwa od razu sugerowała typ, w tym przypadku od razu będzie wiadomo, że to wartość true/false, jeśli nazwa będzie `canDoSth` albo `isSth`.